### PR TITLE
Add nowrap in detail.php (#929)

### PR DESCRIPTION
**Pull request by stefaweb**

### CODE REVIEW

File: mailscanner/style.css [hunk 1]
```diff
@@ -999,3 +999,6 @@ table.mail>tbody>tr>th.alignleft {
     text-align: left;
 }
 
+a {
```
Comment (endelwar) [inline mailscanner/style.css]: Can you limit this css rule to that detail.php specific case?

Comment (stefaweb) [reply mailscanner/style.css]: Hello @endelwar!
A way to to this?

Comment (Skywalker-11) [reply mailscanner/style.css]: You can add a class to the `<a href="..." class="whatever"` and apply it only to that class

Comment (stefaweb) [reply mailscanner/style.css]: Done.

**Pull request merged**